pkg/deploy/devconfig.go

```diff
@@ -174,6 +175,13 @@ func DevConfig(_env env.Core) (*Config, error) {
 			MDMFrontendURL:              ptr.To("https://global.ppe.microsoftmetrics.com/"),
 			MDSDEnvironment:             ptr.To(version.DevGenevaLoggingEnvironment),
 			MsiRpEndpoint:               ptr.To("https://iamaplaceholder.com"),
+			MISEVALIDAUDIENCES: []string{
+				"https://management.core.windows.net/",
+				_env.Environment().ResourceManagerEndpoint,
+			},
+			MISEVALIDAPPIDs: []string{
+				"2187cde1-7e28-4645-9104-19edfa500053",
```

Comment:
can we read this in as config.

Reply:
This is for generating devconfig for full RP deployment, so there is no ARM involved, which means MISE will not be needed. I have added a test detail to make it pass validation.